helm/cilium/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 0.17.0
-appVersion: 1.13.6
+version: 1.14.3
```

Comment:
The `version` field should be updated automatically to the app version on release